### DISCUSSION TITLE
Minima theme had Menu loading issue on Mobile

### DISCUSSION THREAD

**Discussion by nescatfe** [mivinci/hugo-theme-minima#53] [Comments]
I don't really know what's cause the issue, i try to figure it out maybe some script do loop
but you can try it yourself open theme using mobile and try to tap menu, sometimes it's loading too long like the page and loading stuck

here's the video : [https://dropover.cloud/a6ea06](https://dropover.cloud/a6ea06)

**Comment by mivinci** [comment]
You mean like it gets stuck when you navigate back to the tag page from the series page? 🤔

**Reply by nescatfe** [reply]
yes, even from main page to the other page on navbar
I think the issue only happen in chrome browser, tried on safari and it didn't happen

**Reply by mivinci** [reply]
hmmm that is weird 🤔 I've tried on 
Android Chrome and nothing goes wrong. It is probably an issue of iOS Chrome 🤔

**Reply by nescatfe** [reply]
maybe (?) i'll try on android soon

**Comment by nescatfe** [comment]
oh yeah, maybe this is out of topic here but where can i edit or add (?) css for "Tags" in post (not the navbar tags) like [this](https://dropover.cloud/35467a#ebc4524b-9e3a-4664-98a9-68e1e1b821be)
i've been trying edit on main.scss but the social icon css get affect

**Reply by mivinci** [reply]
Sorry but Minima supports no customization for that. If you do want to modify the style of those tags, you can try rewriting or adding your CSS class onto the `<a>` tag [here](https://github.com/mivinci/hugo-theme-minima/blob/a0a89797edb1a72851aad06a56fbd997c158ff80/layouts/_default/single.html#L19).
